src/domain/event/EventOccurrence.tsx

```diff
@@ -56,13 +56,13 @@ const EventOccurrence = ({
   const { childId, eventId } = useParams<UrlParams>();
 
   const date = formatTime(
-    newMoment(occurrence.time),
-    `ddd ${DEFAULT_DATE_FORMAT}`
+    newDate(occurrence.time),
+    `iiiiii ${DEFAULT_DATE_FORMAT}`
```

Comment:
```suggestion
    newDate(occurrence.time),
    `eee ${DEFAULT_DATE_FORMAT}`
```
Would this be a better alternative for the day name?

Reply:
I thought that `iiiiii` was the best fit. The `eee` didn't match with the previous implementation and the `i` is for ISO. I don't know the difference between i and e.

Reply:
Have you tested or do some of the tests have a snapshot that tests the output of moment's `ddd` formatting and date-fns's `iiiiii` formatting? Or is there documentation that would show the format or make it clear what it should output?

Reply:
See https://date-fns.org/v4.1.0/docs/format

Reply:
There was a snapshot test for these. I tried iii, eee and they were wrong. iiiiii was what matched.

Test script (https://stackblitz.com/edit/testing-date-fns-28nwzcqc?file=index.ts): 
```javascript
import format from "date-fns/format";
import { fi } from 'date-fns/locale';
const DATE_FORMAT = "d.M.yyyy";
const TIME_FORMAT = 'HH:mm';
console.log('iiiiii:' + format(new Date(), `iiiiii ${DATE_FORMAT} ${TIME_FORMAT}`, {locale: fi}));
console.log('iii: ' + format(new Date(), `iii ${DATE_FORMAT} ${TIME_FORMAT}`, {locale: fi}));
```

Result:
```
iiiiii:ti 21.1.2025 13:29
iii: tiis. 21.1.2025 13:29
```

Reply:
Here is a script for wider testing. It looks like the `iii` would fit better for English, but `iiiiii` for Finnish. But if they really are ISO-formats, it's a standard then.

``` javascript
import format from 'date-fns/format';
import { fi } from 'date-fns/locale';
const DATE_FORMAT = 'd.M.yyyy';
const TIME_FORMAT = 'HH:mm';

console.log(
  'i: ' +
    format(new Date(), `i ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'ii: ' +
    format(new Date(), `ii ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'iii: ' +
    format(new Date(), `iii ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'iiii: ' +
    format(new Date(), `iiii ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'iiiii: ' +
    format(new Date(), `iiiii ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'iiiiii: ' +
    format(new Date(), `iiiiii ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);

console.log(
  'e: ' +
    format(new Date(), `e ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'ee: ' +
    format(new Date(), `ee ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'eee: ' +
    format(new Date(), `eee ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'eeee: ' +
    format(new Date(), `eeee ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'eeeee: ' +
    format(new Date(), `eeeee ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'eeeeee: ' +
    format(new Date(), `eeeeee ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);

console.log(
  'E: ' +
    format(new Date(), `E ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'EE: ' +
    format(new Date(), `EE ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'EEE: ' +
    format(new Date(), `EEE ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'EEEE: ' +
    format(new Date(), `EEEE ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'EEEEE: ' +
    format(new Date(), `EEEEE ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);
console.log(
  'EEEEEE: ' +
    format(new Date(), `EEEEEE ${DATE_FORMAT} ${TIME_FORMAT}`, { locale: fi })
);

console.log(
  'i: ' +
    format(new Date(), `i ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'ii: ' +
    format(new Date(), `ii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iii: ' +
    format(new Date(), `iii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iiii: ' +
    format(new Date(), `iiii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iiiii: ' +
    format(new Date(), `iiiii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iiiiii: ' +
    format(new Date(), `iiiiii ${DATE_FORMAT} ${TIME_FORMAT}`)
);

console.log(
  'e: ' +
    format(new Date(), `e ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'ee: ' +
    format(new Date(), `ee ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'eee: ' +
    format(new Date(), `eee ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'eeee: ' +
    format(new Date(), `eeee ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'eeeee: ' +
    format(new Date(), `eeeee ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'eeeeee: ' +
    format(new Date(), `eeeeee ${DATE_FORMAT} ${TIME_FORMAT}`)
);


console.log(
  'i: ' +
    format(new Date(), `i ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'ii: ' +
    format(new Date(), `ii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iii: ' +
    format(new Date(), `iii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iiii: ' +
    format(new Date(), `iiii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iiiii: ' +
    format(new Date(), `iiiii ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'iiiiii: ' +
    format(new Date(), `iiiiii ${DATE_FORMAT} ${TIME_FORMAT}`)
);

console.log(
  'E: ' +
    format(new Date(), `E ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'EE: ' +
    format(new Date(), `EE ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'EEE: ' +
    format(new Date(), `EEE ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'EEEE: ' +
    format(new Date(), `EEEE ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'EEEEE: ' +
    format(new Date(), `EEEEE ${DATE_FORMAT} ${TIME_FORMAT}`)
);
console.log(
  'EEEEEE: ' +
    format(new Date(), `EEEEEE ${DATE_FORMAT} ${TIME_FORMAT}`)
);
```

Reply:
Ok, so maybe the documentation was incorrect, or I looked at the wrong version's documentation, or something else. Good that there was a snapshot testing that the content didn't change. Looks ok with this added info.